SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1026,16 +1028,23 @@ void FSpatialGDKEditorToolbarModule::StartInspectorProcess(TFunction<void()> OnR
 {
 	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
 	const FString InspectorVersion = SpatialGDKEditorSettings->GetInspectorVersion();
+	const uint16_t RuntimeGRPCPort = SpatialGDKEditorSettings->GetDefaultPort();
+	// If the Runtime port has changed, the previous Inspector process should be killed 
+	bool bShouldKillInspectorProcess = RuntimeGRPCPort != PreviousRuntimeGRPCPort && PreviousRuntimeGRPCPort != 0;
```

Comment:
Hmmm, was hoping we could use `ULevelEditorPlaySettings::PostEditChangeProperty` in some way but I think this isn't too bad of a workaround to avoid an engine change.

Reply:
Yeah I have tried to avoid an engine change, however, I can change this if you think it is too 'hacky'?